src/libra/jsonrpc/client.py

```diff
@@ -104,6 +104,50 @@ def __init__(
         self._lock = threading.Lock()
         self._retry: Retry = retry or Retry(5, 0.2, StaleResponseError)
 
+    # high level functions
+
+    def get_parent_vasp_account(
+        self, vasp_account_address: typing.Union[libra_types.AccountAddress, str]
+    ) -> rpc.Account:
+        """get parent_vasp account
+
+        accepts child/parent vasp account address, returns parent vasp account
+
+        raise ValueError if given account address is not ChildVASP or ParentVASP account
+        address
+        raise AccountNotFoundError if no account found by given account address, or
+        could not find the account by the parent_vasp_address found in ChildVASP account.
+        """
+
+        account = self.get_account(vasp_account_address)
+        if account is None:
+            hex = utils.account_address_hex(vasp_account_address)
+            raise AccountNotFoundError(f"account not found by address: {hex}")
+
+        if account.role.type == constants.ACCOUNT_ROLE_PARENT_VASP:
+            return account
+        if account.role.type == constants.ACCOUNT_ROLE_CHILD_VASP:
+            return self.get_parent_vasp_account(account.role.parent_vasp_address)
+
+        hex = utils.account_address_hex(vasp_account_address)
+        raise ValueError(f"given account address({hex}) is not a VASP account: {account}")
+
+    def get_account_sequence(self, account_address: typing.Union[libra_types.AccountAddress, str]) -> int:
+        """get on-chain account sequence number
+
+        Calls get_account to find on-chain account information and return it's sequence.
+        Raises AccountNotFoundError if get_account returns None
+        """
+
+        account = self.get_account(account_address)
+        if account is None:
+            hex = utils.account_address_hex(account_address)
```

Comment:
update to use the .hex() function you wrote?

Reply:
actually can't, because the account_address could be string, this all get_account related function accepts hex string and libra_types.AccountAddress as input argument for the account address.